template/src/main/index.js

```diff
@@ -1,23 +1,19 @@
 /* globals INCLUDE_RESOURCES_PATH */
 
 // Set `__resources` path to resources files in main process
-global.__resources = undefined;
-// noinspection BadExpressionStatementJS
-INCLUDE_RESOURCES_PATH
-if(__resources === undefined) console.error('[Main-process]: Resources path is undefined');
+import { app } from 'electron'
 
+// Load here all startup windows
+import './mainWindow'
```

Comment:
Main window should be imported at the end of file (after __resources variable initialization).

Reply:
Just moved the global.__resources into mainWindow.js